src/data/menu/context.menu.config.ts

```diff
@@ -0,0 +1,30 @@
+export const contextMenuConfig = {
```

Comment:
Hmm. Not sure I agree with keeping it as menu config. Can't think of anything right, so I'll merge it and re-architecht later after some thinking :)